File: my-submissions/0075-sort-colors/solution.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n = nums.size();
        for(int i =0;i<nums.size();i++){
            for(int j = i+1;j<n;j++){
                if(nums[i] > nums[j]){
                int temp = nums[i];
                nums[i] = nums[j];
                nums[j] = temp;
                }
            }
        }
    }
};

File: my-submissions/0075-sort-colors/solution.java
class Solution {
    public void sortColors(int[] nums) {
        int[] arr = new int[3];

        for(int i : nums){
            arr[i]++;
        }

            int j = 0;
            int i = 0;
            while(j <3 ){
                while(arr[j] > 0 ){

                nums[i++] = j;
                arr[j]--;
                }
            j++;
            }
        return ;
    }
}
